gateway/src/test/java/ninja/airelemental/GatewayIntegrationTests.java

```diff
@@ -0,0 +1,84 @@
+package ninja.airelemental;
+//
+//import ninja.airelemental.controllers.GatewayUserController;
+//import org.junit.Test;
+//import org.junit.runner.RunWith;
+//import org.springframework.boot.context.embedded.LocalServerPort;
+//import org.springframework.boot.test.context.SpringBootTest;
+//import org.springframework.boot.test.context.SpringBootTest.WebEnvironment;
+//import org.springframework.boot.test.web.client.TestRestTemplate;
+//import org.springframework.http.HttpStatus;
+//import org.springframework.http.ResponseEntity;
+//import org.springframework.test.context.junit4.SpringRunner;
+//
+//import static org.junit.Assert.assertEquals;
+//
+//@RunWith(SpringRunner.class)
+//@SpringBootTest(webEnvironment=WebEnvironment.RANDOM_PORT) // acts like setting server.port property to 0, prevents collisions
+public class GatewayIntegrationTests {
```

Comment:
Next PR should include integration test for Gatewat Controller to test the SimpleGrantedAuthority.

Then we going to focus on the Selenium Tests.